src/components/inputs/upload-input-v2/dropzone.js

```diff
@@ -180,13 +184,21 @@ export class DropzoneJS extends React.Component {
             }
         }
 
-        this.dropzone.on('addedfile', (file) => {
+        this.dropzone.on('addedfile', async (file) => {
```

Comment:
we need to get the token on another method
please do check
https://github.com/dropzone/dropzone/blob/main/src/options.js
https://stackoverflow.com/questions/47208348/dynamic-url-via-http-request-dropzone-js/49901324#49901324